eventing/samples/github-source/README.md

```diff
@@ -0,0 +1,160 @@
+# GitHub Source example
+
+GitHub Source example shows how to wire GitHub events for consumption
+by a Knative Service.
+
+## Deployment Steps
+
+### Prerequisites
+
+You will need:
+
+1. An internet-accessible Kubernetes cluster with Knative Serving
+  installed. Follow the [installation
+  instructions](https://github.com/knative/docs/blob/master/install/README.md)
+  if you need to create one.
+  - Ensure Knative Serving is [configured with a domain
+    name](https://github.com/knative/docs/blob/master/serving/using-a-custom-domain.md)
+    that allows GitHub to call into the cluster.
+  - If you're using GKE, you'll also want to [assign a static IP address](https://github.com/knative/docs/blob/master/serving/gke-assigning-static-ip-address.md).
+1. Install [Knative
+  Eventing](https://github.com/knative/docs/tree/master/eventing).
+1. Create a `Channel`. You can use your own `Channel` or use the
```

Comment:
I think I'd also state, that is this demo uses the in-memory ClusterChannelProvisioner, which needs to be setup:

e.g.
https://github.com/knative/eventing/tree/master/config/provisioners/in-memory-channel#in-memory-channels

Reply:
I did have that in my instructions, but when testing found that the latest eventing releases set that up out of the box. So no extra setup is needed outside of installing Eventing.

Reply:
Ah... ok - good to know, was not aware that's now included!

thanks!